linkcheck/logger/text.py

```diff
@@ -93,7 +93,8 @@ def start_output(self):
 
     def write_intro(self):
         """Log introduction text."""
-        self.writeln(configuration.AppInfo)
+        self.writeln(configuration.App)
+        self.writeln(configuration.Copyright)
```

Comment:
I don't entirely understand the purpose of this change?

Reply:
The copyright line has got rather long. (I saved a line elsewhere, so we are even).

Reply:
Ah, makes sense!